src/polymer/test/PolymerBuildTest.js

```diff
@@ -21,79 +21,124 @@ const deleteFolderRecursive = (path) => {
 }
 
 describe('PolymerBuild', () => {
-  before(() => {
-    if (!fs.existsSync('build')) {
-      fs.mkdirSync('build')
-      fs.mkdirSync('build/bundled')
-    }
-
-    // TODO: Use fs.copyFileSync() after updating to node 8.
-    fs.writeFileSync('htaccess.sample', fs.readFileSync(`${__dirname}/assets/htaccess.sample`))
-  })
+  describe('isValidBaseURI', () => {
+    it('should return true for correct baseURIs', () => {
+      const baseURIs = [
+        '/',
+        '/~ranb2002/',
+        '/www.exemple.com/',
+        'https://www.exemple.com/',
+        'https://www.exemple.com/~ranb2002/',
+      ]
+
+      baseURIs.forEach(baseURI => {
+        const message = `${baseURI} should be a correct baseURI`
+        expect(PolymerBuild.isValidBaseURI(baseURI), message).to.be.true
+      })
+    })
 
-  beforeEach(() => {
-    files.forEach((filename) => {
-      // TODO: Use fs.copyFileSync() after updating to node 8.
-      fs.writeFileSync(`build/bundled/${filename}`, fs.readFileSync(`${__dirname}/assets/${filename}`))
+    it('should return false on missing trailing slash', () => {
```

Comment:
Je mettrait les 2 tests qui retourne false en un seul avec la description `should return false for invalid baseURIs`